test/address.fra.test.js

```diff
@@ -112,6 +112,11 @@ const testcase = (test, common) => {
   assert(`Paris 75000, France`, [
     { locality: 'Paris' }, { postcode: '75000' }, { country: 'France' }
   ])
+
+  // https://github.com/pelias/parser/pull/141#issuecomment-895230721
+  assert(`Esplanade de la Liberté`, [{ street: 'Esplanade de la Liberté' }])
```

Comment:
Thanks for those test cases, I had actually broken the street prefix `Esplanade` for French addresses 😱, but managed to fix them again after adding the examples :+1:

Reply:
:sweat_smile: nice !